.gitlab-ci.yml

```diff
@@ -99,6 +99,7 @@ test:plugin:
     - git fetch --depth=1 origin main
     - rm -rf ~/.gradle/daemon/
     - CODECOV_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android-gradle-plugin.codecov-token --with-decryption --query "Parameter.Value" --out text)
+    - GRADLE_OPTS="-XX:MaxPermSize=512m -Xmx2560m" ./gradlew :dd-sdk-android-gradle-plugin:publishToMavenLocal -Pdd-skip-signing
```

Comment:
Is it necessary?

Reply:
Yes, unfortunately was not able to make it work to get the plugin from the classpath. For some reason it doesn't work with AGP. I am going to give it one more shot today.